### DISCUSSION TITLE
How to delete a Jekyll website to replace with a new one.

### DISCUSSION THREAD

**Discussion by RosarioIacono** [community/community#30072] [Other Feature Feedback, Questions, & Ideas] [answered]
I created my website using the tutorial available at https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll

I used /docs as origin of the data and pushed the website as described.

I successfully created the website rosarioiacono.github.io using Jekyll.

Now I wanted to change the name of the website. To do so, I decided to delete the previous repository and start the process again.

I deleted the repository and removed the origin.

I then started the process trying to create the website datagronomist.io, but at the end, what I get is a website with the address https://rosarioiacono.github.io/datagronomist.github.io/

I would need some help to do that.

**Comment by airtower-luna** [comment, marked as answer]
You only get a `github.io` URL that matches your user or organization name, any other repository name will end up as a subdirectory of your `<username>.github.io` site. If you want that other name you need to either create an organization with that name, or change your username.

Or you could use a custom domain.
